src/coprocessor/dag/expr/builtin_math.rs

```diff
@@ -878,15 +879,35 @@ mod tests {
     #[test]
     fn test_rand_with_seed() {
         let seed: i64 = 20160101;
```

Comment:
Maybe we need to choose the seed randomly ?

Reply:
@iosmanthus @breeswish 
In the `builtin_math`, there are some func that can be test using random, for example in`test_abs`, we can write test like this
```rust
fn test() {
    let test_cnt = 1024;
    let min = -1024 * 1024;
    let max = 1024 * 1024;
    for i in 0..test_cnt {
        let case = random() % (max - min) + min;
        let ans = abs(case);
        if case < 0 {
            assert!(ans + case == 0);
        } else {
            assert!(case == ans);
        }
    }
}
```
so why use stable case to test it.